src/db.c

```diff
@@ -1649,6 +1649,38 @@ int getKeysFromCommand(struct redisCommand *cmd, robj **argv, int argc, getKeysR
     }
 }
 
+/* The method is to extract the channels based on the values in the
+ * pubsublocal related commands.
+ * (first, last, step) - First channel occurence, last channel occurence,
+ * steps between each key.
+ */
+int genericGetChannels(int first, int last, int step,
+                       robj **argv, getKeysResult *result) {
+    UNUSED(argv);
```

Comment:
I'm not clear why you pass in argv if you don't use it. I also think removing it makes the previous line fit into 80 characters :)